grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/ArmeriaClientCall.java

```diff
@@ -453,8 +460,8 @@ public void onNext(DeframedMessage message) {
                 }
             });
         } catch (Throwable t) {
-            final Metadata metadata = new Metadata();
-            close(exceptionHandler.apply(ctx, null, t, metadata), metadata);
+            final Metadata metadata = generateMetadataFromThrowable(t);
```

Comment:
I understood this change means we now always prefer to use `generateMetadataFromThrowable` instead of `new Metadata` if a throwable is provided

Reply:
That is correct. 👍